osCam/osCam/settings.py

```diff
@@ -131,14 +132,13 @@
 LOGIN_REDIRECT_URL = '/'
 
 # Email Notification Settings
-env = environ.Env()
-environ.Env.read_env()
+# environ.Env.read_env()
 
 
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = env('EMAIL_HOST')
-EMAIL_PORT = 587
-EMAIL_USE_TLS = True
-EMAIL_HOST_USER = env('EMAIL_HOST_USER')
-EMAIL_HOST_PASSWORD = env('EMAIL_HOST_PASSWORD')
-RECIPIENT_ADDRESS = env('RECIPIENT_ADDRESS')
+# EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+# EMAIL_HOST = env('EMAIL_HOST')
+# EMAIL_PORT = 587
+# EMAIL_USE_TLS = True
+# EMAIL_HOST_USER = env('EMAIL_HOST_USER')
+# EMAIL_HOST_PASSWORD = env('EMAIL_HOST_PASSWORD')
+# RECIPIENT_ADDRESS = env('RECIPIENT_ADDRESS')
```

Comment:
If we don't need them we can remove them. If they are needed in the future we can still access them through a previous  Commit's hash value

Reply:
ye that seemed overly-complicated, but i do agree in general. 
was going to chat w connor & you abt where this code was gonna live.